modules/heat_conduction/doc/content/source/userobjects/GrayLambertSurfaceRadiation.md

```diff
@@ -0,0 +1,174 @@
+
+  \begin{equation}
+      '~~F_{1,1} F_{1,2} F_{1,3} ;
+       F_{2,1} F_{2,2} F_{2,3} ;
+       F_{3,1} F_{3,2} F_{3,3} ~~'
+  \end{equation}
+
+## The Equations of Gray Lambert Radiators
+
+This section discusses the pertinent equations of the net radiation method and relates them to the
+more fundamental spectral intensity.
+The development of the method is taken from [!cite](modest2013radiative).
+The net radiation method computes the radiative exchange
+between surfaces for the case of no attenuation in the medium between the surfaces. For a point on
+any of the surfaces in radiative heat transfer, a heat balance is formulated at location $\vec{r}$:
+
+\begin{equation}\label{eq:heat_balance}
+  \dot{q}(\vec{r}) = J(\vec{r}) - H(\vec{r}),
+\end{equation}
+
+where $J$ is the radiosity of the surface at point $\vec{r}$ and $H$ is the irradiation from other
+surface to point $\vec{r}$. The units of all quantities in [eq:heat_balance] is $\text{W}/{\text{m}^2}$.
+The radiosity is the sum of the emissive power and the reflected portion of the
+irradiation:
+
+\begin{equation}\label{eq:radiosity}
+  J(\vec{r}) = \epsilon(\vec{r}) \sigma T^4 + \rho(\vec{r}) H(\vec{r}),
+\end{equation}
+
+where $\epsilon$ is the emissivity and $\rho$ is the reflectivity.
+For gray, diffuse surfaces, the reflectivity simply is $\rho = 1 - \epsilon$:
+
+\begin{equation}\label{eq:radiosity_gray}
+  J(\vec{r}) = \epsilon(\vec{r}) \sigma T^4 + (1 - \epsilon(\vec{r})) H(\vec{r}).
+\end{equation}
+
+At this point it is convenient to represent the radiosity, temperature, irradiation,
+and emissivity on each distinct surface $i$ by a suitable average of the actual distribution.
+We leave out the $\vec{r}$ argument and add an $i$ index by which we indicate that this
+quantity has been averaged over the extent of surface $i$.
+
+A second balance equation in addition to [eq:radiosity_gray] is obtained by relating
+the irradiation onto surface $i$ to the radiosities of all other surfaces:
+
+\begin{equation}\label{eq:second_balance}
+  H_i = \sum\limits_{j=1}^n F_{i,j} J_j,
+\end{equation}
+where $A_i$ is the area of surface $i$, $n$ is the total number of surfaces, and $F_{i,j}$ is the view
+factor from surface $i$ to surface $j$. Eliminating $H_i$ from [eq:second_balance] using
+[eq:radiosity_gray] gives:
+
+\begin{equation}\label{eq:fixed_q}
+  \sum\limits_{j=1}^n \left( \delta_{i,j} - F_{i,j }\right) J_j = \dot{q}_i.
+\end{equation}
+
+This result is used for computing $J_j$ on surfaces where $\dot{q}_i$ is known. This is the case for
+`ADIABATIC` surfaces, where $\dot{q}_i = 0$.
+
+A more convenient relationship is derived for surfaces where $T_i$ is either known (`FIXED_TEMPERATURE`) or
+computed (`VARIABLE_TEMPERATURE`). From [eq:radiosity_gray] and [eq:heat_balance], we obtain:
+
+\begin{equation}
+ \dot{q}_i = \frac{\epsilon_i}{1 - \epsilon_i} \left( \sigma T_i^4 - J_i \right).
```

Comment:
An interesting manipulation of the equations. I guess this is related with the constraints on view factors. Can you add a sentence commenting on this?